terraform/cloudfront.tf

```diff
@@ -0,0 +1,49 @@
+resource "aws_cloudfront_distribution" "s3_distribution" {
+  tags = module.beacons_label.tags
+  origin {
+    domain_name            = aws_s3_bucket.backoffice-static.bucket_regional_domain_name
+    origin_id              = aws_s3_bucket.backoffice-static.bucket
+    s3_origin_config {
+      origin_access_identity = aws_cloudfront_origin_access_identity.oai.cloudfront_access_identity_path
+    }
+  }
+
+  enabled             = true
+  default_root_object = "index.html"
+
+  default_cache_behavior {
+    allowed_methods  = ["GET", "HEAD"]
+    cached_methods   = ["GET", "HEAD"]
+    target_origin_id = aws_s3_bucket.backoffice-static.bucket
+
+    forwarded_values {
+      query_string = true
+
+      cookies {
+        forward = "none"
+      }
+    }
+
+    viewer_protocol_policy = "redirect-to-https"
+    min_ttl                = 0
+    default_ttl            = 3600 // 1 hour
+    max_ttl                = 86400 // 1 day
+  }
+
+  price_class = "PriceClass_100"
+
+  restrictions {
+    geo_restriction {
+      restriction_type = "whitelist"
+      locations        = ["GB"]
+    }
+  }
+
+  viewer_certificate {
+    cloudfront_default_certificate = true
+  }
+}
```

Comment:
Do we need to think about adding a more friendly DNS name to the cloudfront host, thinking: dev-backoffice.406beacons.gov.uk?

I am just thinking that we will need a stable DNS name to associate to an MCA DNS label through CNAME, which would be easier to do if the DNS name is fixed rather than being provided by cloudfront which is, I assume, subject to change

Reply:
As discussed on call, ☎️  raising a ticket for this as this isn't a blocker for this PR.

Ticket: https://trello.com/c/Zi3ivd8h/526-create-a-more-friendly-dns-name-to-the-backoffice-cloudfront-host